src/cljs/org/broadinstitute/firecloud_ui/common/style.cljs

```diff
@@ -59,6 +60,9 @@
 (defn create-text-field [props]
   [:input (deep-merge {:type "text" :style input-text-style} props)])
 
+(defn create-search-field [props]
+  [:input (deep-merge {:type "search" :style (assoc input-text-style :WebkitAppearance "none")} props)])
```

Comment:
I didn't try this, but I think this should work as:

    (deep-merge {:type "search" :style {:WebkitAppearance "none"}} props)